pinot-spi/src/main/java/org/apache/pinot/spi/config/table/UpsertConfig.java

```diff
@@ -44,12 +46,16 @@ public enum ConsistencyMode {
   }
 
   @JsonPropertyDescription("Upsert mode.")
-  private Mode _mode;
+  private Mode _mode = Mode.FULL;
 
   @JsonPropertyDescription("Function to hash the primary key.")
   private HashFunction _hashFunction = HashFunction.NONE;
 
+  /// Maintains the mapping of merge strategies per column.
```

Comment:
actually, looks like all comments started with 3 `/` in this file

Reply:
Yeah. 3 `/` means javadoc; 2 `/` is regular comment